client/migrations/sql/tests/cockroach/1_test.sql

```diff
@@ -0,0 +1,4 @@
+-- +migrate Up
```

Comment:
This is updated to latest `13_test.sql`, the only changes I made here are:

Replaced the string `13-` for `1-` and the order of the columns, because I have the `updated_at` and `created_at` columns as the latest.

Reply:
Ah, my bad! I think it would make sense to start migrations with `13_test.sql` and with `13-` so that we can just copy&paste those contents from the other files in the future and to keep consistency across them. What do you think? It would probably also make sense to have `13.sql` instead of `1.sql` in the main migration file, that way it's more coherent across migrations!